malaria24/settings/base.py

```diff
@@ -122,9 +122,22 @@
 DEFAULT_FROM_EMAIL = 'MalariaConnect <malaria24@praekelt.com>'
 
 # JEMBI settings
-JEMBI_URL = 'http://jembi.org/malaria24'
-JEMBI_USERNAME = 'fake@example.com'
-JEMBI_PASSWORD = 'not_a_real_password'
+JEMBI_URL = environ.get('JEMBI_URL') or 'http://jembi.org/malaria24'
+JEMBI_USERNAME = environ.get('JEMBI_USERNAME') or 'fake@example.com'
+JEMBI_PASSWORD = environ.get('JEMBI_PASSWORD') or 'not_a_real_password'
+
+# Onapie settings
+ONAPIE_FORM_PK = environ.get('ONAPIE_FORM_PK') or None
```

Comment:
These `or None`s won't do anything, since `environ.get()` returns `None` if the environment variable isn't set.